emannotationschemas/models.py

```diff
@@ -211,102 +185,69 @@ def split_annotation_schema(Schema):
     return flat_annotation_schema, flat_segmentation_schema
 
 
-def create_segmentation_model(
```

Comment:
this is an example of a function that was used outside that I think you can implement with your new infrastructure, but will minimize the change management if its signature is left unchanged but marked deprecated.
https://github.com/seung-lab/MaterializationEngine/search?q=create_segmentation_model

Reply:
I'm sorry i see you did that I just wasn't expecting it to move in the file.

Reply:
The materialization methods are mainly wrappers for 'make_annotation_model' and 'make_segmentation_model' since in the mat engine we need to read from the metadata to extract the params to pass to the emschemas methods.